src/controllers/reviewController.js

```diff
@@ -0,0 +1,43 @@
+const reviewService = require("../services/reviewService");
+
+const addReview = async (req, res) => {
+  const userId = req.params.id
```

Comment:
미들웨어로 토큰을 검증하는 auth 모듈이 있습니다.

userId는 params로 받는 것이 아니라 req.user와 같이 auth 모듈에서 반환하는 값으로 받으면 좋을 것 같습니다.

다른 controller 함수에서도 마찬가지입니다!

```suggestion
  const userId = req.user.id
```

Reply:
수정완료하였습니다.